assets/components/paymentFailureMessage/paymentFailureMessage.scss

```diff
@@ -10,10 +10,21 @@
 
 }
 
+.component-payment-failure-message--invalid_form_mobile {
+  padding-top: $gu-v-spacing;
+  margin-bottom: -$gu-v-spacing;
+  color: gu-colour(news-garnett-main-1);
+  background-color: gu-colour(news-faded);
+
+  // point at you can see the all the individual invalid field error messages
+  @include mq($from: 361px) {
```

Comment:
do we need to be so aggressive with this value? it seems like it's not too huge an issue if we show it on some larger phones, whereas if we don't show it where it's needed it would be annoying. maybe we could bump it to `phablet`? or at least `mobileMedium`

Reply:
I agree with the principle here, to only show it when you can't see the topmost element error message. The danger is though that as we change the form, this value becomes inaccurate. 

So I think either we do something clever where we detect whether the topmost element is visible, or we play it safer and allow the message to show on larger screens to be safe.

Reply:
Have used `mobile medium`.

Reply:
hey sorry i'm super late to this party but there are vertical media queries (based on screen height) maybe that would be a batter fit for this?